lib/braveAlerter.js

```diff
@@ -0,0 +1,182 @@
+const express = require('express')
+const jsonBodyParser = require('body-parser').json()
+
+const AlertSession = require('./alertSession.js')
+const ALERT_STATE = require('./alertStateEnum.js')
+const AlertStateMachine = require('./alertStateMachine.js')
+const helpers = require('./helpers.js')
+const twilio = require('./twilio.js')
+
+class BraveAlerter {
+    constructor(getAlertSession, getAlertSessionByPhoneNumber, alertSessionChangedCallback, asksIncidentDetails, getReturnMessage) {
```

Comment:
in the BraveAlerter constructor, are `getAlertSession`, `getAlertSessionByPhoneNumber`, and `alertSessionChangedCallback` all function pointers?

Reply:
Yes, that's right. What parameters they should take and what they should return is described in the `README`.